src/templates.c

```diff
@@ -617,7 +617,8 @@ static gchar *run_command(const gchar *command, const gchar *file_name,
 	}
 	else
 	{
-		g_warning("templates_replace_command: %s", error->message);
+		g_warning(_("Cannot execute template replacement command \"%s\": %s. "
+			"Check the path setting in template."), command, error->message);
```

Comment:
something like *Cannot execute command "%s" from the template: %s.  Check the path in the template.* would probably be more correct and easier to understand (as esp. it's not a command that processes templates, but a command replaced in the template).